src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ObjectFormatting/ObjectFormatter.cs

```diff
@@ -16,6 +17,19 @@ internal enum FormatSpecifier
         NoSideEffects = 2
     }
 
+    internal readonly struct ObjectFormatterResult
+    {
+        public string FormattedValue { get; init; }
+
+        public ParameterEvaluationFlags Flags { get; init; }
```

Comment:
Let's have a default value of `ParameterEvaluationFlags.None` instead of requiring everyone to set it? We have to explicitly set this flag to `None` in several places when IMO it should be the default value